src/pages/hub.tsx

```diff
@@ -15,7 +55,58 @@ const MemberHub = () => {
 
   if (isLoading || hasError) return null;
 
-  return <Layout>{user && <section>WELCOME {user?.username}</section>}</Layout>;
+  return (
+    <Layout hideNewsletter>
+      {user && (
+        <section className="py-y-screen-spacing flex flex-col justify-center lg:px-desktop-screen-spacing">
+          <div className="w-full max-w-8xl mx-auto border-1 border-gray-500 rounded-sm px-x-screen-spacing md:px-2 p-2">
+            <img
+              src={coverImage.src}
+              srcSet={coverImage.srcSet}
```

Comment:
I discovered that the `Image` component from `gatsby-image` displays the images with a bad resolution on big screens. So I pick to use an `img` tag instead.

Reply:
I did not add the image on mobile devices because it was looking too small 

![image](https://user-images.githubusercontent.com/20747535/93952736-355f5b80-fd0f-11ea-944c-ab9dfec56a02.png)